evm-adapters/src/call_tracing.rs

```diff
@@ -463,7 +465,11 @@ impl CallTrace {
                     "{}[{}] {}::{}{}({})",
                     left,
                     self.cost,
-                    color.paint(name.unwrap_or(&self.addr.to_string())),
+                    color.paint(
+                        // clippy bug makes us do this
+                        #[allow(clippy::or_fun_call)]
+                        self.label.as_ref().unwrap_or(name.unwrap_or(&self.addr.to_string()))
```

Comment:
lol
curious, does unwrap_or_else make clippy happy?

`unwrap_or_else` would at least be preferable for `name.unwrap_or(&self.addr.to_string())` since this allocates

Reply:
clippy suggests something that isn't valid because it tries to borrow a temporary value when you do or_else